packages/paste-website/src/pages/icon-system/index.mdx

```diff
@@ -0,0 +1,238 @@
+appearance of a web interface and break up the monotony of text. 
+
+Use an icon only when it provides additional clarity or is necessary to draw attention to a UI element.
+If an icon is used to indicate hierarchy of a UI element, all elements at the same hierarchy should also be paired with an icon.
+
+
+## Accessibility
+
+### Specify if an icon is decorative
+
+Icons can either be **decorative** or **informative**. *"Decorative icons don’t add information to the content 
+of a page. The information provided by the image might already be given using adjacent text, or the image 
+might be included to make the website more visually attractive."* (<Anchor href="https://www.w3.org/WAI/tutorials/images/decorative/">w3.org</Anchor>)
+
+Our icons require the developer define whether an icon is decorative by [providing the required property](/icon-system#making-an-icon-decorative-or-informative).
+
+### Give informative icons a title
+
+A title is required so assistive technology can infer equal meaning as a sighted user would.
+
+<DoDont>
+  <Do title="Do" body="Use title text that gives context and meaning to the icon." center>
+    <PlusIcon decorative={false} title="Add to cart" />
+    <Absolute preset="bottom_right">(Hover over icon for tooltip)</Absolute>
+  </Do>
+  <Dont title="Avoid" body="Avoid title text that only describes the icon image." center>
+    <PlusIcon decorative={false} title="Plus" />
+    <Absolute preset="bottom_right">(Hover over icon for tooltip)</Absolute>
+  </Dont>
+</DoDont>
+
+
+### Pair icons with text
+
+Remember that different cultures may interpret the same image in different ways so despite our best intentions it is helpful to provide adjoining text.
+
+<Callout>
+  <CalloutText>
+    We're working on a better example for this section. #TODO
+  </CalloutText>
+</Callout>
+
+<DoDont>
+  <Do title="Do" body="Pair icons with text." center>
+    <PlusIcon decorative={true} />
+    <Box marginLeft="space20">Add to cart</Box>
+  </Do>
+  <Dont title="Avoid" body="Avoid having standalone icons." center>
+    <PlusIcon decorative={false} title="Plus" />
+  </Dont>
+</DoDont>
+
+
+### Icons shouldn't be interactive
+
+Icons should not have interactions such as click or hover behavior. Instead they should be wrapped with an 
+interactive element such as [Button](/components/button) and [Anchor](/components/anchor).
+
+<DoDont>
+  <Do title="Do" body="Wrap icons in an interactive component." center>
+    <Button variant="primary">
+      <PlusIcon decorative={true} />
+      Add to cart
+    </Button>
+  </Do>
+  <Dont title="Avoid" body="Avoid making icons interactive without an interactive metaphor." center>
+    <div onClick={() => {alert(`This isn't a great user experience.`)}}>
+      <PlusIcon decorative={false} title="Add to cart" />
+    </div>
+  </Dont>
+</DoDont>
+
+
+## Examples
+
+#### Adapting the icon's color
+
+We can change the icon color directly using [text color tokens](/tokens/#text-colors).
```

Comment:
Should we phrase this like the Resizing icons text below, i.e.:

We can change the icon color directly using our `textColor` [tokens](/tokens/#text-colors).